src/client/static/index.tsx

```diff
@@ -12,8 +21,12 @@ import './analytics/ophan';
 // initialise google analytics
 gaInit();
 
-// load cmp if it should show
-// use dymamic import to reduce delivered bundle size
-if (shouldShow()) {
-  import('./cmp');
+// don't load this if running in cypress
+if (!window.Cypress) {
```

Comment:
you could use [webpack define plugin](https://webpack.js.org/plugins/define-plugin/) to only output this code if `process.env.NODE_ENV === 'test'` if that's useful